lib/tasks/instrumentation_generator/templates/newrelic.yml.tt

```diff
@@ -0,0 +1,19 @@
+---
+development:
+  error_collector:
+    enabled: true
+  apdex_t: 0.5
+  monitor_mode: true
+  license_key: bootstrap_newrelic_admin_license_key_000
+  instrumentation:
+    <%= @name.downcase %>: <%= $instrumentation_method %>
```

Comment:
Puzzled by the use of global here. I'm curious as well.

Reply:
The global is used in our multiverse suite to swap between `chain` and `prepend`. 

https://github.com/newrelic/newrelic-ruby-agent/blob/1d2016d66e95a2d09c233f6cd2cdae0f696398c2/test/multiverse/lib/multiverse/suite.rb#L634-L636

As mentioned above, it's not working with this template (it's totally blank when `instrumentation:scaffold` is run)

Reply:
Resolved by: c75c8010404817ca14b55798000a97a889efc7dd